src/lib/structures/Event.ts

```diff
@@ -39,15 +39,16 @@ export abstract class Event extends BasePiece {
 		};
 	}
 
-	private async _run(...args: any[]) {
+	private async _run(...args: unknown[]) {
 		try {
+			// @ts-expect-error Argument of type 'any[]' is not assignable to parameter of type 'E extends string | number ? ClientEvents[E] : any[]'. (2345)
```

Comment:
Is this still needed?

Reply:
Yep since it checks if it's a ClientEvents entry and adds the types otherwise just unknown. Tested it and it works as hoped for.

Reply:
Should update the message accordingly since it's now `unknown[]`

Reply:
Oh I didn't notice it was directed at the comment not the code.